core-blocks/audio/index.js

```diff
@@ -57,10 +63,10 @@ export const settings = {
 	edit,
 
 	save( { attributes } ) {
-		const { autoplay, caption, loop, src } = attributes;
+		const { autoplay, caption, loop, preload, src } = attributes;
 		return (
 			<figure>
-				<audio controls="controls" src={ src } autoPlay={ autoplay } loop={ loop } />
+				<audio controls="controls" src={ src } autoPlay={ autoplay } loop={ loop } preload={ preload } />
```

Comment:
This will need a deprecated `save` function, right? (https://github.com/WordPress/gutenberg/blob/master/docs/block-api/deprecated-blocks.md)

Or will this load fine because it's just adding the binary attrs?

Reply:
The latter, to my knowledge. The markup doesn't change structure; this is only a progressive enhancement.

Reply:
Oh right, cool. 👍 (Sorry I'm still a bit new to block validation!)